swan-lake/integration-tutorials/content-based-message-routing.md

```diff
@@ -243,77 +257,13 @@ Follow the instructions given in this section to develop the service.
-        - The `doctor` value is assigned to the `doctor` variable of string type.
-        - The remaining components of the payload are collected in a mapping value and assigned to the `reservationRequest` variable.
-
-    - The `log` functions are used to [log](https://ballerina.io/learn/by-example/#log) information at `INFO`, `DEBUG`, and `ERROR` log levels.
-
     - Use a [match statement](https://ballerina.io/learn/by-example/match-statement) to assign the relevant client to the `hospitalEP` variable based on the `hospital_id` value.
 
-        ```ballerina
-            match hospital_id {
-                GRAND_OAK => {
-                    hospitalEP = grandOakEP;
-                }
-                CLEMENCY => {
-                    hospitalEP = clemencyEP;
-                }
-                _ => {
-                    hospitalEP = pineValleyEP;
-                }
-            }
-        ```
-
     - Make a `POST` request to the backend hospital service to make the reservation. The `category` value is used as a path parameter.
 
-        ```ballerina
-        ReservationResponse|http:ClientError resp = hospitalEP->/[category]/reserve.post({
-            patient,
-            doctor,
-            ...reservationRequest
-        });
-        ```
-
-        The expected payload in the request to make the reservation consists of four fields, namely `patient`, `doctor`, `hospital`, and `appointment_date`. The `reservationRequest` variable is used as a rest argument since it contains `hospital`, and `appointment_date`. The payload is specified directly as an argument to the remote method call.
-
-        ```ballerina
-        {patient, doctor, ...reservationRequest}
-        ```
-
     - Use the `is` check to check whether the response is a `ReservationResponse` and return it as is (i.e., reservation successful).
 
-        ```ballerina
-        if resp is ReservationResponse {
-            log:printDebug("Reservation request successful",
-                            name = patient.name,
-                            appointmentNumber = resp.appointmentNumber);
-            return resp;
-        }
-        ```
-
-    - If the response is not a `ReservationResponse`, log the failure at `ERROR` level.  Return a "NotFound" response if the response is a `http:ClientRequestError`, or an "InternalServerError" response if the response is a `http:ServerError`.
-
-        ```ballerina
-        log:printError("Reservation request failed", resp);
-        if resp is http:ClientRequestError {
-            return <http:NotFound> {body: "Unknown hospital, doctor or category"};
-        }
-
-        return <http:InternalServerError> {body: resp.message()};
-        ```
+    - If the response is not a `ReservationResponse`, log the failure at `ERROR` level.  Return a "NotFound" response if the response is an `http:ClientRequestError`, or an "InternalServerError" response if the response is an `http:ServerError`.
```

Comment:
If the response is not a `ReservationResponse` record (i.e., `http:ClientError`), which indicates that the request failed, log the failure at `ERROR` level.